traits/tests/test_trait_list_object.py

```diff
@@ -93,6 +93,13 @@ def test_defaults(self):
         self.assertEqual(event.removed, [])
         self.assertEqual(event.added, [])
 
+    def test_trait_list_event_str_representation(self):
```

Comment:
How about adding a test for a `TraitListEvent` subclass, to check that it's the subclass name that appears in the `repr` rather than `TraitListEvent`.  (That's the original motivation for the change, after all.)